docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -27,148 +29,227 @@ This guide provides steps for creating a Docker image with Intel® Distribution
-        --remove Microsoft.VisualStudio.Component.Windows81SDK || IF "%ERRORLEVEL%"=="3010" EXIT 0 && powershell set-executionpolicy remotesigned
-   ```
-
-   In case of proxy issues, please use the [offline installer for Build Tools](https://docs.microsoft.com/en-us/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019).
-
-### <a name="config-image-for-gpu"></a>Configuring the Image for GPU
-
-> **NOTE**: Since GPU is not supported in <a href="#get-prebuilt-image">prebuilt images</a> or [default Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles), you must make sure the Additional Requirements for GPU in <a href="#system-requirements">System Requirements</a> are met, and do the following steps to build the image manually.
-
-1. Reuse one of [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles). You can also use your own Dockerfile. 
-2. Check your [Windows host and container isolation process compatibility](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
-3. Find the appropriate Windows container base image on [DockerHub](https://hub.docker.com/_/microsoft-windows) and set up your host/container version in the `FROM` Dockerfile instruction.  
-   For example, in the `openvino_c_dev_<version>.dockerfile`, change:  
-   ```bat
-   FROM mcr.microsoft.com/windows/servercore:ltsc2019 AS ov_base
-   ```
+
+   .. code-block:: sh
+      RUN powershell.exe -Command Invoke-WebRequest -URI https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile %TMP%\\vs_buildtools.exe
+
+      RUN %TMP%\\vs_buildtools.exe --quiet --norestart --wait --nocache `
+            --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools" `
+            --add Microsoft.VisualStudio.Workload.MSBuildTools `
+            --add Microsoft.VisualStudio.Workload.UniversalBuildTools `
+            --add Microsoft.VisualStudio.Workload.VCTools --includeRecommended `
+            --remove Microsoft.VisualStudio.Component.Windows10SDK.10240 `
+            --remove Microsoft.VisualStudio.Component.Windows10SDK.10586 `
+            --remove Microsoft.VisualStudio.Component.Windows10SDK.14393 `
+            --remove Microsoft.VisualStudio.Component.Windows81SDK || IF "%ERRORLEVEL%"=="3010" EXIT 0 && powershell set-executionpolicy remotesigned
+
+   In case of proxy issues, use the [offline installer for Build Tools](https://docs.microsoft.com/en-us/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2019).
+
+.. tab:: Configuring the Image for GPU
+
+   .. note::
+      Since GPU is not supported in prebuilt images or [default Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles), you must make sure the Additional Requirements for GPU in <a href="#system-requirements">System Requirements</a> are met, and do the following steps to build the image manually.
```

Comment:
```suggestion
      Since GPU is not supported in prebuilt images or [default Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles), make sure the Additional Requirements for GPU in <a href="#system-requirements">System Requirements</a> are met, and do the following steps to build the image manually.
```